Formula/bgpq4.rb

```diff
@@ -0,0 +1,26 @@
+class Bgpq4 < Formula
+  desc "BGP filtering automation for Cisco, Juniper, BIRD and OpenBGPD routers"
+  homepage "https://github.com/bgp/bgpq4"
+  url "https://github.com/bgp/bgpq4/archive/refs/tags/0.0.7.tar.gz"
+  sha256 "c39af7d2a8b0f4cf61543e776677baaf067aa087181851572211d168258f7f9f"
+  license "BSD-2-Clause"
+  head "https://github.com/bgp/bgpq4.git"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "autoconf"
+  depends_on "automake"
+
+  def install
+    system "./bootstrap"
```

Comment:
Does this download anything?

Reply:
No, it just runs autoconf/automake: https://github.com/bgp/bgpq4/blob/main/bootstrap